### PR TITLE
Parameter sweeps polish

### PR DESCRIPTION
**Pull request by augustehirth**
Some phrasing changes. Ran a formatter. Cleared outputs in Simulation

### CODE REVIEW

File: docs/params.ipynb [hunk 1]
```diff
@@ -102,10 +102,10 @@
    "source": [
     "q0 = cirq.LineQubit(0)\n",
     "\n",
-    "circuit1 = cirq.Circuit([cirq.H(q0), cirq.Z(q0)**0.5, cirq.H(q0), cirq.measure(q0)])\n",
+    "circuit1 = cirq.Circuit([cirq.H(q0), cirq.Z(q0) ** 0.5, cirq.H(q0), cirq.measure(q0)])\n",
```
[the remark this answers is not in the document]
Comment (augustehirth) [reply docs/params.ipynb:105]: I used black and black[jupyter]. Running `black --version` results in `black, 22.3.0 (compiled: yes)`. Should I downgrade?

[the remark this answers is not in the document]
Comment (augustehirth) [reply docs/params.ipynb:105]: OK to leave it as-is for now then?

File: docs/params.ipynb [hunk 2]
```diff
@@ -499,7 +510,7 @@
     "id": "756ad9a92137"
    },
    "source": [
-    "Notice that the new circuit has new symbols, `<a/2>` and `<1-a/2`. These are not expressions.  We can see this by looking at the value of the exponent in the first gate:"
+    "Notice that the new circuit has new symbols, `<a/2>` and `<1-a/2`, which are explicitly not expressions.  You can see this by looking at the value of the exponent in the first gate:"
```
Comment (maffoo) [inline docs/params.ipynb:513]:
```suggestion
    "Notice that the new circuit has new symbols, `<a/2>` and `<1-a/2>`, which are explicitly not expressions.  You can see this by looking at the value of the exponent in the first gate:"
```

File: docs/simulation.ipynb [hunk 3]
```diff
@@ -639,11 +530,11 @@
     "id": "10617793bfd2"
    },
    "source": [
-    "Here we see that the `Symbol` is used in two gates, and then the resolver\n",
+    "In the previous example, he `Symbol` `x` is used in two gates, and then the resolver\n",
```
Comment (maffoo) [inline docs/simulation.ipynb:533]:
```suggestion
    "In the previous example, the symbol `x` is used in two gates, and then the resolver\n",
```

File: docs/simulation.ipynb [hunk 4]
```diff
@@ -639,11 +530,11 @@
...
     "provides this value at run time.\n",
     "\n",
-    "Parameterized values are most useful in defining what we call a\n",
-    "`sweep`.  A `sweep` is a sequence of trials, where each\n",
+    "Parameterized values are most useful in defining what is called a\n",
+    "`sweep`, which is a sequence of trials, where each\n",
```
Comment (maffoo) [inline docs/simulation.ipynb:537]:
```suggestion
    "\"sweep\", which is a sequence of trials, where each\n",
```

File: docs/simulation.ipynb [hunk 5]
```diff
@@ -639,11 +530,11 @@
...
     "trial is a run with a particular set of parameter values.\n",
     "\n",
     "Running a `sweep` returns a `Result` for each set of fixed parameter\n",
```
Comment (maffoo) [inline docs/simulation.ipynb:540]:
```suggestion
    "Running a sweep returns a `Result` for each set of fixed parameter\n",
```

**Review by maffoo — APPROVED** [review]
Minor comments, then LGTM. Up to you if you'd like to revert the change to whitespace around `**`.

**Pull request merged**